Algorithm.CSharp/RenkoConsolidatorAlgorithm.cs

```diff
@@ -25,6 +25,7 @@ namespace QuantConnect.Algorithm.CSharp
     /// </summary>
     /// <meta name="tag" content="renko" />
     /// <meta name="tag" content="indicators" />
+    /// <meta name="tag" content="wicked"/>
```

Comment:
nit - unrequired change?

Reply:
This helps our documentation search for the types